### DISCUSSION TITLE
ability to add files or folders during the converting

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#84] [Ideas]
It would be really nice if he had the ability to add files or folders  to the batch during the converting. possibly a queue

**Comment by nagadomi** [comment]
I take this as a request for a more advanced scheduler feature.
Currently, you can specify a folder or text file (list of images/videos) with the same settings.

Implementing a queue that is enabled only during program running time is not difficult.
Saving/restoring the schedule is complicated.

**Comment by nagadomi** [comment]
This needs more GUI windows.
A list of jobs, a viewer for each job, etc.
I have developed a job scheduler before, so I can implement it, but it is not an interesting task, so this is low on my priority list.

**Reply by loawizard** [reply]
Ok ! 
Just a quick question. Can you add files to the converter folder after you started a conversion (if you pick a folder)?

**Reply by nagadomi** [reply]
Currently, files added after the start are not detected.
The feature to detect additional files can be easily supported. However, it is not possible to change the processing order, so they are processed last.
